mw-webapp/src/model/firebaseCollection/DayReport.ts

```diff
@@ -29,9 +31,9 @@ export class DayReport {
   public problemsForCurrentPeriod: string[];
 
   /**
-   * Anything that student wants to say about work
+   * Student comments
    */
-  public studentComments: string[];
```

Comment:
Victor told don't repeat info. He was mean don't write types of properties (arrays, strings etc) But I say about explain what is studentComment 
@tritonJS826 Do I understand wrong your position?

Reply:
"Document only useful information that couldn't be placed in code (or if it is hard etc.).
in current case Student comment will be enough (yeah code is self-documented in this case)"

Reply:
It's good if everyone understand thr same what you student can write in the cell studendComments/ But even now we have long discuss because of misunderstanding. So without explain what is inside studentComment discuss will be in the future too

Reply:
Fully deleted this model